deploy/crds/app.kiegroup.org_v1alpha1_kogitoinfra_cr.yaml

```diff
@@ -10,3 +10,4 @@ spec:
   installKafka: false
   # set to true if a Keycloak instance should be installed in the namespace via Keycloak Operator
   installKeycloak: false
+status: {}
```

Comment:
Just a curiosity, were you able to to an oc import on the files where the status was added?
I got an issue yesterday while adding the keycloack crds, had to remove it [1] to work.

[1] - https://github.com/keycloak/keycloak-operator/blob/master/deploy/crds/keycloak.org_keycloaks_crd.yaml#L13-L14

Reply:
This was add because the `KogitoInfra` doesn't have a default status (https://issues.redhat.com/browse/KOGITO-995). In the case you linked, it's because its a subresource, should be sent differently to the API. In this case, you should use the status API and not sending everything together (like oc does).